Tasks/month.js

```diff
@@ -15,12 +17,13 @@ Months = [
   'dec',
 ];
 
-Month = (s) => {
-  l = Months.length;
-  for (let i = 0; i < l; i++) {
-    if (s.toLowerCase().startsWith(Months[i])) return i + 1;
+const getMonthNumber = (month) => {
```

Comment:
```suggestion
const getMonthNumber = month => {
```
As for me, it looks better without brackets (  ), but both are good!

Reply:
We have common eslint config for that